src/FaunaDBSchemaProvider.ts

```diff
@@ -42,7 +42,7 @@ export default class FaunaDBSchemaProvider
       | FunctionSchemaItem
   ): Thenable<vscode.TreeItem[]> {
     if (element instanceof DBSchemaItem || !element) {
-      const dbPath = element ? `${element.name}` : undefined;
+      const dbPath = element?.itemPath;
```

Comment:
@BrunoQuaresma Some TypeScript power right here 💪One more benefit of using TS, since JS still doesn't have this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
yes! I love this feature.